cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -61,7 +61,7 @@ func deletePermissiveNodesBindingWhenUsingNodeAuthorization(client clientset.Int
 
 	nodesRoleBinding.Subjects = newSubjects
 
-	if _, err := client.RbacV1beta1().ClusterRoleBindings().Update(nodesRoleBinding); err != nil {
+	if _, err := client.RbacV1().ClusterRoleBindings().Update(nodesRoleBinding); err != nil {
```

Comment:
if this is targeting 1.9, you can actually delete this... the nodes role binding isn't created any longer in 1.8+

Reply:
done

Reply:
Yeah, I know. However, that's coming as an other, future PR